### DISCUSSION TITLE
Sequent Microsystems 8-Relay v3 id's not matching actual relays

### DISCUSSION THREAD

**Discussion by tagyoureit** [tagyoureit/nodejs-poolController#431] [REM] [answered]
I remember seeing something a while ago that the lights on the 8-relay hat didn't match the relay that was on.  However, I'm seeing that the relays themselves are not in the right order/assignment.  Is anyone else seeing this?  

For example, when I turn on Relay 7 on REM, it turns on the physical relay 5.  I thought maybe something was awry with REM, but I see the same thing via command line.  
Eg `8relay 0 write 5 1` turns on relay 7 and the relay 7 light.  

This is the entire mapping I see on my board:
```
Relay ID 1 = physical relay 1
Relay 2 = 2
Relay 3 = 8
Relay 4 = 7
Relay 5 = 3
Relay 6 = 4
Relay 7 = 5
Relay 8 = 6
```
Only relays 1 and 2 map to the physical locations and ID's.

**Comment by rstrouse** [comment, marked as answer]
I asked them about it and the screen printing is wrong.  The print was moved to the front side of the board but they never re-arranged the leds.  Also make sure you have the IND controller selected.
